samples/addons/prometheus.yaml

```diff
@@ -127,6 +127,13 @@ data:
         - __address__
         - __meta_kubernetes_service_annotation_prometheus_io_port
         target_label: __address__
+      - action: replace
```

Comment:
This file is generated from Prometheus helm chart (manifests/addons/values-prometheus.yaml). Can we fix it upstream? Or if it is, just update the version we use

Reply:
Prometheus Community has update the configuration to match ipv6 (https://github.com/prometheus-community/helm-charts/pull/2127/files). Maybe this file should have similar update?

Reply:
The only accepted changes are in manifests/addons/gen.sh or manifests/addons/values-prometheus.yaml. The file being modified is generated from those

Reply:
Thanks. I understand how to change this file now. Prometheus helm-charts version should be 15.9.3 or higher to support IPV6.